src/sentry/api/serializers/models/organization.py

```diff
@@ -116,43 +116,13 @@ def serialize(self, obj, attrs, user):
         }
 
 
-class DetailedOrganizationSerializer(OrganizationSerializer):
+# Does not include project/teams list
+class LightDetailedOrganizationSerializer(OrganizationSerializer):
```

Comment:
I'd almost rather call this `DetailedOrganizationSerializer` and rename the other one to `DetailedOrganizationSerializerWithTeams` (or something like `WithRelations` or w/e)

Reply:
That would match the naming of our `Team` serializer